chain-tx-enclave/enclave-ra/ra-sp-server/src/ias_client.rs

```diff
@@ -0,0 +1,142 @@
+use ra_common::{sp::AttestationEvidence, AttestationReport};
+use reqwest::blocking::Client;
+use thiserror::Error;
+
+/// Client used for connecting to Intel Attestation Service (IAS)
+pub struct IasClient {
+    /// IAS API Key
+    ias_key: String,
+    /// HTTP client
+    http_client: Client,
+    /// Base URI of intel attestation service (IAS)
+    ias_base_uri: String,
+    /// API path to get SigRL from IAS
+    ias_sig_rl_path: String,
+    /// API path to get attestation report from IAS
+    ias_report_path: String,
+}
+
+impl IasClient {
+    /// Creates a new instance of IAS client
+    pub fn new(
+        ias_key: String,
+        ias_base_uri: String,
+        ias_sig_rl_path: String,
+        ias_report_path: String,
+    ) -> Self {
+        let http_client = Client::new();
+
+        Self {
+            ias_key,
+            http_client,
+            ias_base_uri,
+            ias_sig_rl_path,
+            ias_report_path,
+        }
+    }
+
+    /// Gets SigRL (Signature revocation list) from IAS
+    pub fn get_sig_rl(&self, gid: [u8; 4]) -> Result<Option<Vec<u8>>, IasClientError> {
```

Comment:
how about
get_signature_revocation_list

Reply:
All the names corresponds to Intel IAS API Specifications. Intel calls these "SigRL". So, I gess, it is best to stay true to specs.